app/models/booksuggestion.rb

```diff
@@ -1,4 +1,4 @@
-class BookSuggestion < ApplicationRecord
-  validates :title, :author, :link, presence: true
+class Booksuggestion < ApplicationRecord
```

Comment:
After making the changes requested for the last PR, you can leave this like it was